README.asciidoc

```diff
@@ -15,7 +15,14 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ////
 == TinkerPop3
-image::https://codecov.io/gh/apache/tinkerpop/branch/master/graph/badge.svg?token=TojD2nR5Qd[link="https://codecov.io/gh/apache/tinkerpop"]
+image:https://img.shields.io/maven-central/v/org.apache.tinkerpop/gremlin-driver?color=brightgreen[link="https://mvnrepository.com/artifact/org.apache.tinkerpop/gremlin-driver"]
```

Comment:
This links are to the latest 3.6.2 version, may be better to also add a link to the 3.5.5?

Reply:
From what I've seen I agree most projects tend to just link the latest version. As it's configured now, the badges will all update once a new release is published. It is possible to have it link to different versions depending on what branch you are in but I fear that may involve manually updating the badges as part of the release process (the badge providers I'm using here don't seem to support such functionality).

My preference would be to leave it with just the latest version unless there is enough demand for branch based versioning to justify the additional setup and maintenance.

Reply:
Agreed